### ISSUE TITLE
Plex PIN API certificate signed by unknown authority

### ISSUE THREAD

**Issue by CarterBrehm** [aunefyren/wrapperr#89]
Trying to prop this service back up for the holiday season and running into an issue. The x509 certificate error, except it's to a service that I know has a good cert: Plex itself. 

```
2023/12/21 14:10:10 Timezone set to America/Chicago.
2023/12/21 14:10:10 Running version v3.2.4.
2023/12/21 14:10:10 Starting Wrapperr on port 8282.
2023/12/21 14:10:10 Starting using HTTP.
[GIN] 2023/12/21 - 14:10:12 | 200 |    1.026735ms |      172.20.0.1 | GET      "/"
[GIN] 2023/12/21 - 14:10:12 | 200 |    1.188007ms |      172.20.0.1 | GET      "/assets/css/wrapped.css"
[GIN] 2023/12/21 - 14:10:12 | 200 |     132.868µs |      172.20.0.1 | GET      "/manifest.json"
[GIN] 2023/12/21 - 14:10:12 | 200 |       537.7µs |      172.20.0.1 | GET      "/js/service-worker.js"
[GIN] 2023/12/21 - 14:10:12 | 200 |     518.057µs |      172.20.0.1 | GET      "/js/functions.js"
[GIN] 2023/12/21 - 14:10:12 | 200 |     898.746µs |      172.20.0.1 | GET      "/js/index.js"
[GIN] 2023/12/21 - 14:10:12 | 200 |     837.736µs |      172.20.0.1 | GET      "/js/get_stats.js"
[GIN] 2023/12/21 - 14:10:12 | 200 |     582.964µs |      172.20.0.1 | GET      "/js/get_functions.js"
[GIN] 2023/12/21 - 14:10:13 | 200 |     2.14819ms |      172.20.0.1 | GET      "/assets/img/gift.svg"
[GIN] 2023/12/21 - 14:10:13 | 200 |    3.597902ms |      172.20.0.1 | GET      "/assets/loading.gif"
[GIN] 2023/12/21 - 14:10:13 | 200 |     522.552µs |      172.20.0.1 | GET      "/assets/external-link.svg"
[GIN] 2023/12/21 - 14:10:13 | 200 |      300.75µs |      172.20.0.1 | GET      "/assets/done.svg"
[GIN] 2023/12/21 - 14:10:13 | 200 |     381.953µs |      172.20.0.1 | GET      "/assets/close.svg"
[GIN] 2023/12/21 - 14:10:13 | 200 |     391.375µs |      172.20.0.1 | GET      "/assets/trash.svg"
2023/12/21 14:10:13 Retrieved Wrapperr version. - Origin: 172.20.0.1
[GIN] 2023/12/21 - 14:10:13 | 200 |    3.187315ms |      172.20.0.1 | POST     "/api/get/wrapperr-version"
[GIN] 2023/12/21 - 14:10:13 | 200 |    1.641552ms |      172.20.0.1 | GET      "/assets/img/favicons/favicon.ico"
[GIN] 2023/12/21 - 14:10:13 | 200 |    7.511861ms |      172.20.0.1 | GET      "/assets/winter.webp"
[GIN] 2023/12/21 - 14:10:15 | 404 |       3.167µs |      172.20.0.1 | GET      "/apple-touch-icon-precomposed.png"
[GIN] 2023/12/21 - 14:10:15 | 404 |       1.459µs |      172.20.0.1 | GET      "/apple-touch-icon.png"
[GIN] 2023/12/21 - 14:10:15 | 404 |       2.104µs |      172.20.0.1 | GET      "/favicon.ico"
[GIN] 2023/12/21 - 14:10:15 | 200 |    1.167779ms |      172.20.0.1 | GET      "/assets/img/favicons/favicon.ico"
[GIN] 2023/12/21 - 14:10:19 | 200 |     317.088µs |      172.20.0.1 | GET      "/?close_me=true"
[GIN] 2023/12/21 - 14:10:19 | 200 |     173.893µs |      172.20.0.1 | GET      "/manifest.json"
2023/12/21 14:10:19 Failed to retrieve Plex Auth pin. Error: Get "https://plex.tv/api/v2/pins/105104808": tls: failed to verify certificate: x509: certificate signed by unknown authority
[GIN] 2023/12/21 - 14:10:19 | 500 |  319.882418ms |      172.20.0.1 | POST     "/api/login/plex-auth"
[GIN] 2023/12/21 - 14:11:23 | 200 |     306.815µs |      172.20.0.1 | GET      "/"
[GIN] 2023/12/21 - 14:11:23 | 200 |     158.255µs |      172.20.0.1 | GET      "/manifest.json"
```

Is there any way I can remove the certificate check from this function? I can't use Plex OAuth, but the rest of the application works perfectly.

**Comment by aunefyren** [comment]
Are you on Docker? I am experiencing this too. I think the newest slim version of the image is missing a package needed for TLS. 

If this is the case, use v3.2.3 until a hotfix is out.

**Comment by Kyrluckechuck** [comment]
I'm seeing the same thing!
Reverting to `v3.2.3` resolved it for now, and that theory seems to check out 😛

**Comment by aunefyren** [comment]
Fixed in v3.2.5.
